examples/advanced-react-app/src/App.jsx

```diff
@@ -0,0 +1,166 @@
+import { useEffect, useState } from 'react';
+import useDCPWorker from 'use-dcp-worker';
+
+const paymentAddress = import.meta.env.VITE_WORKER_PAYMENT_ADDRESS;
+
+function App() {
+  const [workerPaymentAddress, setWorkerPaymentAddress] = useState(paymentAddress);
```

Comment:
**nitpick:** It's a little confusing having the examples swap between what the default/initial value is, and what the state variable is for configuration.